nixos/modules/services/networking/networkmanager.nix

```diff
@@ -263,7 +263,12 @@ in {
            { source = overrideNameserversScript;
              target = "NetworkManager/dispatcher.d/02overridedns";
            }
-      ++ lib.imap1 (i: s: {
+    ++ optional (cfg.basePackages ? networkmanager_l2tp)
+    {
+      target = "ipsec.d/stub";
+      text = "";
```

Comment:
here I just want to create the ipsec.d folder but didn't know how to do it without creating a stub file.